ClassesHomework/Dog.cs

```diff
@@ -10,9 +10,9 @@ namespace ClassesHomework
     class Dog : Animal
     {
         //properties/variables for the class
-        //public string  Breed     { get; set; }
-        //public int     Age       { get; set; }
-        //public bool    IsSitting { get; set; }
+        //public string Breed      { get; private set; }
```

Comment:
This would be correct, but the properties have been moved out of Dog and into its parent class. That's where the change will need to be made.